### ISSUE TITLE
Error in installation

### ISSUE THREAD

**Issue by Quantizen** [aziezahmed/podcast-player#3]
```
> pip install podcast-player

Collecting podcast-player
  Downloading podcast-player-1.1.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/1n/kzz_j7g519d482trw8xfx4xh0000gn/T/pip-build-tnvgW3/podcast-player/setup.py", line 9, in <module>
        from podcast_player import __version__
      File "podcast_player/__init__.py", line 2, in <module>
        from .podcast_database import PodcastDatabase
      File "podcast_player/podcast_database.py", line 3, in <module>
        from sqlobject import *
    ImportError: No module named sqlobject
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/1n/kzz_j7g519d482trw8xfx4xh0000gn/T/pip-build-tnvgW3/podcast-player/
```
I both try `Python2 & 3`, the same error.

**Comment by aziezahmed** [comment]
I am able to replicate that where I don't have the python sqlobject installed.
I have listed sqlobject as a dependency, so in theory it should install before installing podcast-player.

I'm going to need to further investigate why this is happening, but in the meantime you could try installing sqlobject yourself first.

`$ pip install sqlobject`

followed by:

`$ pip install podcast-player`

**Comment by Quantizen** [comment]
Is it possible to add [IINA](https://lhc70000.github.io/iina/) as the player? It also has MPV as its core.

**Comment by aziezahmed** [comment]
As long as you can launch iina from the terminal you should be able to set the player with the set-player command:
`podcast set-player mpv`

Assuming that is the case you could try:
`podcast set-player iina`

However, if iina can't launch from the terminal like that you could set it similar to this command on macOS:
`podcast set-player "open -a /Applications/QuickTime\ Player.app"`
This launches the podcast in Quicktime.

**Comment by aziezahmed** [comment]
This issue should now be fixed with version 1.2.0
Closing Issue.
